vignettes/Introduction_To_BiocParallel.Rmd

```diff
@@ -0,0 +1,867 @@
+---
+title: "**Introduction to *BiocParallel***"
+author:
+- name: "Valerie Obenchain"
+- name: "Vincent Carey"
+- name: "Michael Lawrence"
```

Comment:
I think you should add your name after this line; use the `affiliation:` tag as a 'hack' to generate a footnote indicating what you contributed
```
- name: "Phylis Atieno"
  affiliation: "Vignette translation from Sweave to Rmarkdown / HTML"